src/revChatGPT/__main__.py

```diff
@@ -1,8 +1,11 @@
-from revChatGPT.revChatGPT import Chatbot
+from __future__ import annotations
```

Comment:
Is this a dependency I need to add to the requirements? I don't know much about annotations sorry

Reply:
Let me test everything works. These were changes made purely from running test and I didn't personally touch the code. I will keep you updated on the running portion.

Reply:
To clarify the change here it looks like it is following [PEP-563](https://peps.python.org/pep-0563/)

Reply:
> To clarify the change here it looks like it is following [PEP-563](https://peps.python.org/pep-0563/)

Seems to be a built in library. I won't worry about it then

Reply:
I see the issue all though it is accurate it is not needed I had a arg from another project of mine in the config still.

Reply:
If it's unnecessary please remove it. Minimal dependencies are recommended for the main code.